contents/blog/measuring-feature-success.md

```diff
@@ -0,0 +1,130 @@
+---
+title: How we measure feature success at PostHog
```

Comment:
```suggestion
title: How we build features users love (really fast)
```

Reply:
@ivanagas Thoughts on this as a headline? Something about the phrase "feature success" just felt clunky to me. This speaks to me! Agree?